layers/b2qt/recipes-qt/automotive/neptune3-ui/neptune.service

```diff
@@ -12,6 +12,7 @@ Environment=LC_ALL=en_US
 Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/root/dbus/user_bus_socket
 Environment=QT_IM_MODULE=qtvirtualkeyboard
 Environment=XDG_RUNTIME_DIR=/tmp
+Environment=QTWEBENGINE_CHROMIUM_FLAGS="--no-sandbox --single-process"
```

Comment:
If --single-process is used for all targets then remove its inclusion for only rpi in 
https://github.com/Pelagicore/meta-pelux/blob/master/layers/b2qt/recipes-qt/automotive/neptune3-ui_git.bbappend

Remove 
FORCE_SINGLE_PROCESS = "${@bb.utils.contains('MACHINE', 'raspberrypi3', ' --force-single-process', '', d)}" line and fix the recipe accordingly.
unless the --single-process for QTWEBENGINE_CHROMIUM_FLAGS works differently from the --single-process commandline argument for neptune-ui

Reply:
From what i have tested.. the `QTWEBENGINE_CHROMIUM_FLAGS` environment variable, and the actual neptune service arguments are different from each other. Therefore the `--force-single-process` and `QTWEBENGINE_CHROMIUM_FLAGS="--single-process"` are different.

Reply:
Can confirm what @fhajredini states.

Reply:
This (that they are different and that we do X for rpi alredy) is a perfect example of something worth mentioning in the commit message IMHO. Would:

1.  Make it easier for reviewers. Tariq would not have had to ask about it. Others who did not know about it are informed.

2. Make it more clear what is going on when looking at this in the future. People will not have to relearn the hard way what you already learned. Will even future Fisnik remember the details about this a year from now?

Please update the commit message and try to think more about putting useful information like this in future commit messages.

(Would also be nice to have more details as to why they are different. Might require looking at e.g. the Neptune code though... do not have to go there... I guess...)

/Commi message police signing off. :)